src/plugins/console/server/lib/proxy_request.test.ts

```diff
@@ -83,14 +91,7 @@ describe(`Console's send request`, () => {
     });
 
     // Set a host header
-    const result2 = await proxyRequest({
-      agent: null as any,
-      headers: { Host: 'myhost' },
-      method: 'get',
-      payload: null as any,
-      timeout: 30000,
-      uri: new URL('http://noone.nowhere.none'),
-    });
+    const result2 = await sendProxyRequest({ headers: { Host: 'myhost' } });
```

Comment:
[nit]: Clarify naming
```suggestion
    const resultWithHostHeader = await sendProxyRequest({ headers: { Host: 'myhost' } });
```